src/frontend/screens/Invites/AlreadyOnProject.tsx

```diff
@@ -0,0 +1,97 @@
+import {BottomSheetWrapper} from '../../sharedComponents/BottomSheetWrapper';
+import {StyleSheet, View} from 'react-native';
+import {PrimaryButton, SecondaryButton} from '../../sharedComponents/Buttons';
+import {defineMessages, useIntl} from 'react-intl';
+import {NativeRootNavigationProps} from '../../sharedTypes/navigation';
+import {HeaderText} from '../../sharedComponents/Text/HeaderText';
+import {BodyText} from '../../sharedComponents/Text/BodyText';
+import Error from '../../images/Error.svg';
+import {useProjectSettings} from '@comapeo/core-react';
+import {useActiveProjectId} from '../../contexts/ActiveProjectIdStoreContext';
+import {useListenToInviteStateUpdate} from '../../hooks/useListenToInviteStateUpdate';
+
+const m = defineMessages({
+  leaveProj: {
+    id: 'screens.Invite.AlreadyOnProject.leaveProj',
+    defaultMessage: 'Leave Current Project',
+  },
+  goBack: {
+    id: 'screens.Invite.AlreadyOnProject.goBack',
+    defaultMessage: 'Go Back',
+  },
+  alreadyOnProject: {
+    id: 'screens.Invite.AlreadyOnProject.alreadyOnProject',
+    defaultMessage: 'You are already on a project',
+  },
+  onProject: {
+    id: 'screens.Invite.AlreadyOnProject.onProject',
+    defaultMessage: 'You are on {projectName}',
+  },
+  leaveWarning: {
+    id: 'screens.Invite.AlreadyOnProject.leaveWarning',
+    defaultMessage: 'To join a new project you must leave your current one.',
+  },
+});
+export const AlreadyOnProject = ({
+  route,
+  navigation,
+}: NativeRootNavigationProps<'AlreadyOnProject'>) => {
+  const {formatMessage} = useIntl();
+  const projectId = useActiveProjectId();
+  const {
+    data: {name: currentProjectName},
+  } = useProjectSettings({projectId: projectId!});
```

Comment:
this can/should be using `useActiveProject()` which guarantees the existence of the `projectId` i.e.

```suggestion
  const {projectId} = useActiveProject();
  const {
    data: {name: currentProjectName},
  } = useProjectSettings({projectId});
```

a bit confusing for now but one of mentioned items to clean up in #1081